### PR TITLE
5574: add to helm charts

### PR DESCRIPTION
**Pull request by talawson05**
added covid feature flag to helm charts

### CODE REVIEW

File: local-helm/local-overrides.yaml [hunk 1]
```diff
@@ -46,6 +46,8 @@ mp:
 
 pb: 
   enabled: true  
+  featureFlags:
```
Comment (stuart-bass-cgi) [inline local-helm/local-overrides.yaml]: can you please either undo this change, or bring in all the feature flags so that it is consistent?

**Review by stuart-bass-bjss — APPROVED** [review]
